fortinet_fortigate/plugin.spec.yaml

```diff
@@ -685,24 +685,26 @@ actions:
     title: Create Address Object
     description: Create an address object
     input:
-      ip:
-        title: IP
-        description: IP
+      host:
+        title: Host
+        description: This can be an IP address, a CIDR or a domain name e.g. 198.51.100.100, 192.0.2.0/24, example.com
```

Comment:
```suggestion
        description: The host to create. This can be an IP address, a CIDR or a domain name e.g. 198.51.100.100, 192.0.2.0/24, example.com
```